lib/kino_db/connection_cell.ex

```diff
@@ -227,6 +233,23 @@ defmodule KinoDB.ConnectionCell do
     end
   end
 
+  defp to_quoted(%{"type" => "duckdb"} = attrs) do
+    var = quoted_var(attrs["variable"])
+
+    quote do
+      :ok = Adbc.download_driver!(:duckdb)
+
+      adbc_db_conf =
+        case unquote(attrs["database_path"]) do
+          "" -> {Adbc.Database, driver: :duckdb}
+          path -> {Adbc.Database, driver: :duckdb, path: path}
+        end
+
+      {:ok, db} = Kino.start_child(adbc_db_conf)
+      {:ok, unquote(var)} = Kino.start_child({Adbc.Connection, database: db})
+    end
+  end
```

Comment:
We want the generation itself to be conditional, like this:

```suggestion
  defp to_quoted(%{"type" => "duckdb"} = attrs) do
    var = quoted_var(attrs["variable"])
    
    opts =
      if path = attrs["database_path"] do
        [driver: :duckdb, path: path]
      else
        [driver: :duckdb]
      end

    quote do
      :ok = Adbc.download_driver!(:duckdb)

      {:ok, db} = Kino.start_child({Adbc.Database, unquote(opts)})
      {:ok, unquote(var)} = Kino.start_child({Adbc.Connection, database: db})
    end
  end
```

(Please test that it works :))

Reply:
sure, it works as expected, but attrs["database_path"] could be an empty, falling always in the first clause, I had to use case to check for empty string

Reply:
Oh yeah, that should be the check definitely!